src/App.vue

```diff
@@ -52,48 +45,6 @@
       attach="#app"
       @close="notInGoodStandingDialog = false"
     />
-
-    <!-- Loading Dashboard spinner -->
-    <v-fade-transition>
-      <div
-        v-show="showLoadingDashboardSpinner"
```

Comment:
You can delete this getter.

And also "spinnerText" below.

And any getters they call if no longer used.